addon/components/frost-scroll.js

```diff
@@ -42,28 +53,28 @@ export default Component.extend(PropTypeMixin, {
   _setupPerfectScroll () {
     const debouncePeriod = 150
 
-    scheduleOnce('afterRender', this, () => {
+    run.scheduleOnce('afterRender', this, () => {
```

Comment:
OK, I suppose it's not evil, although I did find it kinda ugly and hard to follow, but I don't believe it's in line with the reason why we destructure (I keep calling it deconstructing) the `Ember` object. At least, to the best of my memory, we were doing it b/c we wanted to mimic (and make it easy to switch to) named exports. You can't destructure a named export, so I don't think we should use nested destructuring of `Ember`. You can use nested destructuring of one of the named exports after the import line, so I wouldn't be opposed to use having a second line that uses nested destructuring if we want to go that route, but the main reason I didn't like the nested on the main `Ember` line was b/c it makes it so that now, it's a lot more work to convert to named exports, probably not automatable like it would be if we stuck with first destructuring the top-level attributes in one statement, then any other ones in follow-up statements.